countries-of-the-world/main.js

```diff
@@ -0,0 +1,61 @@
+async function sortCountries() {
+	try {
+		const result = await axios.get('https://restcountries.eu/rest/v2/all');
+		const sortedCountries = result.data.sort(
+			(country1, country2) => country1.population - country2.population
+		);
+		for (country of sortedCountries) {
+			const { name, flag, region } = country;
```

Comment:
Tof dat je de spread operator gebruikt. Deze had je ook op regel 7 binnen de for-loop kunnen plaatsen